internal/querynode/impl_test.go

```diff
@@ -320,6 +320,33 @@ func TestImpl_isHealthy(t *testing.T) {
 	assert.True(t, isHealthy)
 }
 
+func TestImpl_ShowConfigurations(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	etcdCli, err := etcd.GetEtcdClient(&Params.EtcdCfg)
+	assert.NoError(t, err)
+	defer etcdCli.Close()
+
+	var wg sync.WaitGroup
+
+	wg.Add(1)
```

Comment:
wait group for what? It seems no other goroutines

Reply:
I will add some other goroutines for test the failed condition, but I do not know if wait group is necessary either.

Reply:
wg is not needed here since parallel run is not enabled